packages/dds/merge-tree/package.json

```diff
@@ -32,6 +32,7 @@
 		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
 		"postpack": "tar -cvf ./merge-tree.test-files.tar ./src/test ./dist/test",
 		"perf": "cross-env FLUID_TEST_VERBOSE=1 mocha \"dist/**/*.spec.js\" --node-option unhandled-rejections=strict,expose-gc --exit -r node_modules/@fluidframework/mocha-test-setup --perfMode --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js --timeout 30000",
+		"perf:profile": "node --inspect-brk ./node_modules/mocha/bin/mocha.js \"dist/**/*.spec.js\" --node-option unhandled-rejections=strict,expose-gc --exit -r node_modules/@fluidframework/mocha-test-setup --perfMode --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js --timeout 30000",
```

Comment:
does this add warnings to fluid-build / other output? as long as it doesn't i think it's fine to leave here. there's probably a better way to do it though (i'm just familiar with this flow so when it's necessary I add it)

Reply:
Can that be verified just by running the `npm run build` command for merge-tree? If so, there don't seem to be new warnings. Or is `fluid-build` a tool for doing something else?

Reply:
check `build:fast` from the root

Reply:
i looked at the implementation just now though. doesn't look like it should complain (it only checks standardized scripts).